playground/db/src/index.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * A Workers KV-based database for storing shareable code snippets in the Playground.
+ */
+
+export interface Env {
+  PLAYGROUND: KVNamespace;
+}
+
+export default {
+  async fetch(
+    request: Request,
+    env: Env,
+    ctx: ExecutionContext,
+  ): Promise<Response> {
+    const headers = {
+      "Access-Control-Allow-Origin": "*",
+    };
+
+    switch (request.method) {
```

Comment:
Could you add some documentation about the supported URL schema? Or / and refactor out the common path (getting the `PLAYGROUND` env variable and doing some sort of slicing on the pathname (this is where an inline comment would be useful, what's the thing we remove? 

Could we use `url.hash`? (assuming this is what you're slicing off)